### PR TITLE
Bugfix/login

### PR DESCRIPTION
**Pull request by wavyllama**
Add Username/Password verification, Deleting of users from CLI, and storing of messages.

### CODE REVIEW

File: server/session_test.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 	"testing"
 )
 
-const(
+const (
 	bobDisplayName = "Bobby B"
```
Comment (wavyllama) [inline server/session_test.go]: Just saw this :+1:

File: db/friend.go [hunk 2]
```diff
@@ -25,9 +25,9 @@ func addFriend(username, displayName, macAddress, ipAddress, friendUsername stri
 	return ExecuteChangeCommand(insertCommand, "Failed to add friend")
 }
 
-// Delete a friend by their MAC Address
-func deleteFriend(username, macAddress string) bool {
-	deleteCommand := fmt.Sprintf("DELETE FROM %s WHERE username=\"%s\" AND friend_mac_address= \"%s\"", friendsTableName, username, macAddress)
+// Delete a friend by their displayName Address
```
Comment (sameetandpotatoes) [inline db/friend.go]: NP: Should be displayName, not displayName address.

File: server/server.go [hunk 3]
```diff
@@ -293,7 +295,9 @@ func (s *Server) SendChatMessage(friendDisplayName, message string) error {
 	(*chatMsg).Text = cyp[0]
 
 	chatMsg.NewPayload(s.User.MAC, s.User.IP, s.User.Username, friend.Username)
-	(*chatMsg).Text = []byte(message)
+	bytes := []byte(message)
+	(*chatMsg).Text = bytes
```
Comment (sameetandpotatoes) [inline server/server.go]: Since I made a comment already, can you just do this in one line?

File: server/session_test.go (hunk 1 above)
Comment (sameetandpotatoes) [reply server/session_test.go]: Thanks, I knew you would get it 👍

File: server/server.go (hunk 3 above)
Comment (wavyllama) [reply server/server.go]: I would rather not i use it twice and its kindof ugly

File: db/friend.go (hunk 2 above)
Comment (wavyllama) [reply db/friend.go]: I will change this

File: server/server.go (hunk 3 above)
Comment (sameetandpotatoes) [reply server/server.go]: Fine.

**Pull request merged**